bin/skyfactory/select_and_merge_samples.py

```diff
@@ -229,7 +229,7 @@ def pair_files(ofiles, tfiles, pzfiles):
                 nz_cut = [99, 0.0]
 
             if sample == 'LSS':
-                def cut_fcn(o, t, p, sys_map_vals, zcol): LSS_cuts(o, t, p,
+                def cut_fcn(o, t, p, sys_map_vals, zcol): return LSS_cuts(o, t, p,
```

Comment:
If not cut_fcn will be None, and cause some error at the line of mask &= cut_fcn